vmdb/app/models/classification.rb

```diff
@@ -176,17 +176,18 @@ def self.create_category!(options)
   def self.categories(region_id = self.my_region_number, ns = DEFAULT_NAMESPACE)
     result = []
     if region_id
-      cats = Classification.in_region(region_id).find(:all, :conditions => "classifications.parent_id = 0", :include => [:tag, :children])
+      cats = Classification.in_region(region_id).where("classifications.parent_id = 0").includes(:tag, :children)
     else
-      cats = Classification.find(:all, :conditions => "classifications.parent_id = 0", :include => [:tag, :children])
+      cats = Classification.where("classifications.parent_id = 0").includes(:tag, :children)
     end
     cats.each { |c| result.push(c) if c.tag2ns(c.tag.name) == ns }
     result
   end
 
   def self.category_names_for_perf_by_tag(region_id = self.my_region_number, ns = DEFAULT_NAMESPACE)
-    find_opts = {:conditions => {:parent_id => 0, :perf_by_tag => true}, :include => [:tag]}
-    self.in_region(region_id).all(find_opts).collect { |c| c.name if c.tag2ns(c.tag.name) == ns }.compact
+    self.in_region(region_id).
+      where(:parent_id => 0, :perf_by_tag => true).
+      includes(:tag).collect { |c| c.name if c.tag2ns(c.tag.name) == ns }.compact
```

Comment:
Ah, I didn't do it... but I'll fix it.  Must be that Seattle coffee.  cough @tenderlove